recipes/nb_anacondacloud/yum_requirements.txt

```diff
@@ -0,0 +1 @@
+fontconfig
```

Comment:
I think we have fontconfig  in conda Forge.  Is there a reason you need to yum install it? 

https://github.com/conda-forge/fontconfig-feedstock